site/src/utilities.ts

```diff
@@ -259,7 +259,9 @@ function getPrerequisiteFormatHtml(
       .map((childPrereq) =>
         getPrerequisiteFormatHtml(priorCourses, childPrereq, false)
       )
-      .join(` ${prereq.type} `);
+      .join(
+              prereq.type == "and" ? " and "+(topLevel ? "<br />" : "") : " or "
```

Comment:
This is a pretty gnarly nested ternary - if this logic is kept, it should be broken into a more readable function.